schema_editor/app/scripts/views/sidebar/sidebar-partial.html

```diff
@@ -8,18 +8,11 @@
                     <li><a ui-sref="rt.add">Add a new event type</a>
                 </ul>
             </li>
-            <li class="active-parent in">
-                <a>&#8212; Crimes</a>
-                <ul>
-                    <li><a ui-sref="rt.detail({uuid: 'abcd'})">View related content</a></li>
-                    <li><a ui-sref="rt.schema-add({uuid: 'abcd'})">Add related content</a></li>
-                </ul>
-            </li>
-            <li class="active-parent in">
-                <a>&#8212; Accidents</a>
+            <li class="active-parent in" ng-repeat="recordType in sb.recordTypes | orderBy:'label'" class="row">
```

Comment:
There's also a small bit of functionality where clicking the header with the label collapses/expands the list. 

All that needs to be done for that to work is to toggle one of classes:`active-parent in` on click and leave that class off by default. I just don't remember which one.